pyelftools should be required

```
  File "/home/libkrunfw-3.12.0/bin2cbundle.py", line 173, in <module>                                                          
    sys.exit(main())
             ^^^^^^                                                                                                                     
  File "/home/libkrunfw-3.12.0/bin2cbundle.py", line 158, in main                                                              
    load_addr = write_elf_cbundle(ifile, ofile)                                                                                         
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                         
  File "/home/libkrunfw-3.12.0/bin2cbundle.py", line 31, in write_elf_cbundle                                                  
    elffile = ELFFile(ifile)                                                                                                            
              ^^^^^^^                                                                                                                   
NameError: name 'ELFFile' is not defined. Did you mean: 'elffile'?                                                                      
make: *** [Makefile:78: kernel.c] Error 1        
```

--- a/bin2cbundle.py
+++ b/bin2cbundle.py
@@ -1,9 +1,7 @@
-try:
-    from elftools.elf.elffile import ELFFile
-except:
-    pass
 import argparse
 import sys
+
+from elftools.elf.elffile import ELFFile
 
 # Use 64k page size for rounding. This should cover 4k/16k/64k kernels
 PAGE_SIZE = 65536

